core/jobs.py

```diff
@@ -33,8 +33,8 @@
 from mapreduce import context
 from mapreduce import input_readers
 from mapreduce import mapreduce_pipeline
-from mapreduce import model as mapreduce_model
 from mapreduce import output_writers
+from mapreduce import model as mapreduce_model
```

Comment:
/cc @apb7 -- can you work with @vojtechjelinek on resolving this lint issue? It keeps cropping up.

Reply:
Hi @vojtechjelinek, we recently changed the import-order in this file. Please follow the steps as stated in this [discussion](https://groups.google.com/forum/#!topic/oppia-dev/mVzwx0TDip0)
The steps are as follows:

1. Run bash scripts/clean.sh
2. Run bash scripts/start.sh
3. Run python scripts/pre_commit_linter.py --path=./core/jobs.py
4. `isort` should now complain about the error locally. Pull the `develop` branch once again. This would correct the import order and the error would be resolved.

(@seanlip: I think this would solve the issue).

Reply:
Done.